src/INFO.sh

```diff
@@ -35,7 +35,7 @@ cat <<EOF
 package="omniedge"
 version="${VERSION}"
 arch="${PLATFORMS}"
-description="Connect all your devices using WireGuard, without the hassle."
```

Comment:
Hi @brucebot, could you please help verify if this an accurate description for omniedge?